journal/updated/05-chapter-4.adoc

```diff
@@ -39,205 +39,205 @@ Bristol, Fourth month 30th, 1763.
-with our dear friend Elizabeth Bevan visited several families,
-being engaged to labour among them in spirit,
-and in the demonstration and power of God to preach the Gospel;
-for indeed the Gospel is the power of God, which is often preached in our souls;
+with our dear friend Elizabeth Bevan, we visited several families,
+being engaged to labour among them in spirit
+and in the demonstration and power of God to preach the Gospel.
+For indeed the Gospel is the power of God, which is often preached in our souls;
 and did all that hear it live in obedience unto it,
-they would know redemption from sin and come to witness a being reconciled unto God,
-his Spirit bearing witness with their spirits that they are his children.
+they would know redemption from sin and come to witness a reconciliation unto God,
+His Spirit bearing witness with their spirits that they are His children.
 
-On the 24th we proceeded to Carmarthen, and lodged at William Reynolds`'s,
-where we had a meeting: from there to Haverfordwest to John Lewis`'s,
+On the 24th we proceeded to Carmarthen and lodged at William Reynolds`'s,
+where we had a meeting. From there we travelled to Haverfordwest, to John Lewis`'s,
 who went with us over a large and difficult ferry to Jamestown, where we had a meeting.
-On first-day attended meetings at Haverfordwest,
+On first-day we attended meetings at Haverfordwest,
 and in the evening had an opportunity in the house
-of a Friend at which several were present;
-it was an exercising season,
-but I was at length favoured with strength to speak in a very close manner, and to query,
+of a Friend at which several were present.
+It was an exercising season,
+but I was at length favoured with strength to speak in a very close manner, and to query
 whether the fathers and mothers, who once had been serviceable in the church of God,
 were not now laying waste what they had helped to build!
-It seemed to me to be so;
+It seemed to me to be so,
 and that those who should have been the repairers
 of breaches had broken down the wall of discipline,
-and made way for the destruction of their children;
-yet that the Lord had mercifully visited them with his servants,
-and had hewn them by his prophets, but if they would not return and repent,
-I believed the Lord would remove the stumbling blocks and pluck up the dead trees;
-and in place thereof, raise up a people whom he would purify,
-and that such should tread his courts, and only such could sing his praise.
-By thus discharging my duty, although it was in fear, I obtained peace;
+and made way for the destruction of their children.
+Yet the Lord had mercifully visited them with His servants,
+and had hewn them by His prophets; but if they would not return and repent,
+I believed the Lord would remove the stumbling blocks and pluck up the dead trees,
+and in the place thereof, raise up a people whom He would purify,
+and those such should tread His courts, and only such could sing His praise.
```

Comment:
The word "those" that you added here doesn't seem to fit. Maybe switch it back to just "and such should". Such here kinda means "those" or "these" or "such as these should tread his courts..."